string auction_type
string task_name
string task_data
int64 auction_ID

int64 creator_ID
string creator_name

string winner_name
int64 winner_ID
